src/guide/introduction.md

```diff
@@ -128,7 +128,7 @@ export default {
 </script>
 
 <template>
-  <button @click="count++">Count is: {{ count }}</button>
+  <button @click="count++">Count: {{ count }}</button>
```

Comment:
I understand why you're removing the `is`, but for the sake of the semantics of being explicit, let's not change this.